android-components/components/service/nimbus/src/main/java/mozilla/components/service/nimbus/messaging/NimbusMessagingStorage.kt

```diff
@@ -54,10 +54,17 @@ class NimbusMessagingStorage(
     /**
      * Returns the [Message] for the given [key] or returns null if none found.
      */
+    suspend fun getMessage(key: String): Message? = createMessage(key)
```

Comment:
drive by - what benefit does having the extra `getMessage` func have over just using a public `createMessage` ?

Reply:
Oh good spot. I've changed this to re-use the accept a `featureValue`.